src/ape/api/providers.py

```diff
@@ -921,7 +935,22 @@ def start(self, timeout: int = 20):
         else:
             logger.info(f"Starting '{self.process_name}' process.")
             pre_exec_fn = _linux_set_death_signal if platform.uname().system == "Linux" else None
-            self.process = _popen(*self.build_command(), preexec_fn=pre_exec_fn)
+
+            self.base_logs_path.mkdir(parents=True, exist_ok=True)
+            if self.stdout_logs_path.is_file():
+                self.stdout_logs_path.unlink()
+            if self.stderr_logs_path.is_file():
+                self.stderr_logs_path.unlink()
```

Comment:
```suggestion
            self.stdout_logs_path.unlink(missing_ok=True)
            self.stderr_logs_path.unlink(missing_ok=True)
```

Reply:
> 

we can't actually do this yet because of python 3.7

Reply:
Ah yes. That's a shame :)